spec/jobs/searchgov_url_fetcher_job.rb

```diff
@@ -0,0 +1,30 @@
+require 'spec_helper'
+
+describe SearchgovUrlFetcherJob do
+  subject(:perform) { SearchgovUrlFetcherJob.perform_now(args[:searchgov_url]) }
+  let!(:searchgov_url) { SearchgovUrl.create(url: 'https://agency.gov/') }
+  let(:args) do
+    { searchgov_url: searchgov_url }
+  end
+
+  it_behaves_like 'a searchgov job'
+
+  describe '#perform' do
+    it 'must have one parameter' do
+      expect{ SearchgovUrlFetcherJob.perform_now }.
+        to raise_error(ArgumentError)
+    end
+
+    it 'fetches a searchgov_url' do
+      # byebug
+      perform
+      expect(searchgov_url.last_crawl_status).to_not be(nil)
```

Comment:
We don't really need the record to be fetched in our test, nor do we need the test to know anything about what `fetch` does (such as changing the `last_crawl_status`). A cleaner (and faster) way to test this would be to use a mock, i.e.:
```ruby
expect(searchgov_url).to receive(:fetch)
perform
```

Reply:
The change you made is what I was looking for 👍